script/benchmark

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+function benchmark() {(
+  cd tmp/bench-site
+  bundle update
+  time bundle exec jekyll build
+)}
+
+rm -rf tmp/benchmark tmp/bench-site
+git clone --depth 1 https://github.com/jekyll/benchmarking.git tmp/benchmark
+
+( cd tmp/benchmark; ./generate_site.sh 1000 ../bench-site )
```

Comment:
./generate_site.sh: 2: ./generate_site.sh: Syntax error: "(" unexpected

Reply:
I don’t know what this is about. It works on my local Bash.
I’ll keep playing.